File: C/Fibonacci.c
#include<stdio.h>
    int main()
    {
        int a=0,b=1,i,c,z;
        printf("Enter range : ");
        scanf("%d",&z);
        printf("%d\t%d\t",a,b);
        for(i=1;i<=z;i++)
        {
            c=a+b;
        	printf("%d\t",c);
            a=b;
            b=c;                        
        }
        return 0;
    }

File: Python/Industrial Training/fibonacci.py
# Fibonacci series
n = int(input("Enter range : "))
if n > 0:
    a, b = 0, 1
    if n == 1:
        print(a,end=' ')
    if n >= 2:
        print(a,b,end=' ')
    for i in range(3,n+1):
        c = a + b
        print(c,end=' ')
        a = b
        b = c
else:
    print("Invalid input")